domain/consensus/processes/dagtraversalmanager/dagtraversalmanager_test.go

```diff
@@ -111,3 +208,52 @@ func TestLowestChainBlockAboveOrEqualToBlueScore(t *testing.T) {
 		checkExpectedBlock(tipHash, 10, blueScore11BlockHash)
 	})
 }
+
+func createAChainDag(genesisHash *externalapi.DomainHash, tc testapi.TestConsensus) (*externalapi.DomainHash, error) {
+	block := genesisHash
+	var err error
+	for i := 0; i < 5; i++ {
+		block, _, err = tc.AddBlock([]*externalapi.DomainHash{block}, nil, nil)
+		if err != nil {
+			return nil, err
+		}
+	}
+	return block, nil
+
+}
+func expendDagToTwoChildrenSameSelectedParent(parentHash *externalapi.DomainHash, tc testapi.TestConsensus) (*externalapi.DomainHash, *externalapi.DomainHash, error) {
+
+	firstChild, _, err := tc.AddBlock([]*externalapi.DomainHash{parentHash}, nil, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	secondChild, _, err := tc.AddBlock([]*externalapi.DomainHash{parentHash}, nil, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	return firstChild, secondChild, nil
+
+}
+
+func expendDagToDiffChains(firstChainHash, secondChainHash *externalapi.DomainHash, tc testapi.TestConsensus) (*externalapi.DomainHash, *externalapi.DomainHash, error) {
+	block := firstChainHash
+	var err error
+	for i := 0; i < 3; i++ {
```

Comment:
Note that here you hard-coded 3 and 2, while `checkExpectedBlockOnTwoDiffChain` receives those.
I'd suggest defining constants for these.

Similar problem you have at `createAChainDag`